### ISSUE TITLE
Replace deprecated MongoDB methods

### ISSUE THREAD

**Issue by mzch** [muety/anchr#8]
when I treid to add a new link to collection, an error occurred as below:

```
[2019-06-11 10:03:43.856] [ERROR] defaultLogger - RES: 203.141.134.112 POST /api/collection/Q
5Fga/links 5cffd9e449a8f92230e6fd56 500 Unknown modifier: $pushAll MongoError: Unknown modifi
er: $pushAll
    at Function.MongoError.create (/srv/anchr/node_modules/mongodb-core/lib/error.js:31:11)
    at toError (/srv/anchr/node_modules/mongoose/node_modules/mongodb/lib/utils.js:114:22)
    at /srv/anchr/node_modules/mongoose/node_modules/mongodb/lib/collection.js:985:67
    at /srv/anchr/node_modules/mongodb-core/lib/topologies/server.js:780:13
    at Callbacks.emit (/srv/anchr/node_modules/mongodb-core/lib/topologies/server.js:95:3)
    at Connection.messageHandler (/srv/anchr/node_modules/mongodb-core/lib/topologies/server.js:249:23)
    at Socket.<anonymous> (/srv/anchr/node_modules/mongodb-core/lib/connection/connection.js:265:22)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
```

I had the same error on both Mongo 3.6 and Mongo 4.0.10

**Comment by mzch** [comment]
p.s

Since 3.6, $pushAll is deprecated.

**Comment by n1try** [comment]
Thanks for that catch. Feel free to open a PR to replace deprecated methods.

**Comment by muety** [comment]
Should be fixed with 519e16da2a7ce364f309ed0a0d859505156b06f4.
